NEWS.md

```diff
@@ -1,5 +1,11 @@
 # spatialsample (development version)
 
+* `spatial_buffer_vfold_cv()` has had some attribute changes to match `rsample`:
+  * `strata` attribute is now the name of the column used for stratification, 
+     or FALSE if there was no stratification.
```

Comment:
My question here is the same. It looks like `NULL` plus column works, and that's a better option than making an attribute that might be a logical or a string.